src/main/java/ch/njol/skript/classes/data/DefaultConverters.java

```diff
@@ -255,6 +259,14 @@ public DefaultConverters() {}
 //			}
 //		});
 
+		// String - Statistic
+		Converters.registerConverter(String.class, Statistic.class, statistic -> {
+			try {
+				return Statistic.valueOf(statistic.toUpperCase(Locale.ENGLISH));
+			} catch (Exception ex) {
+				return null;
+			}
+		});
```

Comment:
I'm not a fan of adding String converters at all. This allows Skript to see this converter and attempt against it. I would suggest using Regex parsing exactly like how World classinfo deals with parsing Worlds. This will be less attention grabbing for Skript.

(This is separate from the String to World converter, as that shouldn't exist either as the ClassInfo handles quotations.)